kobo/hub/xmlrpc/worker.py

```diff
@@ -234,7 +234,7 @@ def get_awaited_tasks(request, awaited_task_list):
 
 
 @validate_worker
-def create_subtask(request, label, method, args, parent_id):
+def create_subtask(request, label, method, args, parent_id, subtask_priority):
```

Comment:
Because there is no default value for the new argument, this is a backwards-incompatible change which would require the hub and the worker to be upgraded in lockstep.  Can this please be avoided?  Letting it default to `None` should be good enough.

Also, could you please share some general info on how you are using subtasks?  I ask this because I've not been aware of anyone using the feature before, which leaves open the possibility that the functionality is completely broken.

Reply:
Thanks @kdudka for explanation. @rohanpm the code has been updated according to your review.